### ISSUE TITLE
Social Icons not visible

### ISSUE THREAD

**Issue by huteck** [pmoreno-rodriguez/grav-theme-future2021#34]
The Social Icons are not visible while being in html code (see picture) but with a size of 0. (Both in Sidebar mode as well as in normal footer). I also tried to add the social icon in the header to cross-check but there one out of the two sizes is 0 as well.
This is on a fresh Grav installation without any plugins or changes to the template.

![image](https://user-images.githubusercontent.com/117048767/223561560-46873ab2-4426-4286-a11e-353ab2025749.png)

**Comment by pmoreno-rodriguez** [comment]
Could you send an online version, where can I see it?

**Comment by pmoreno-rodriguez** [comment]
I've just install a fresh Grav in my localhost folder, wit composer and I've installed the Future2021 with gpm. This is my future2021.yaml (previously I've copied it from theme folder to config/themes folder):
```
enabled: true
production-mode: false
enable_social: true
social:
  -
    name: Twitter
    url: 'https://www.twitter.com'
    icon: 'fa fa-twitter'
  -
    name: Facebook
    url: 'https://www.facebook.com'
    icon: 'fa fa-facebook'
  -
    name: Instagram
    url: 'https://www.instagram.com'
    icon: 'fa fa-instagram'
```
All works fine

![future2021_footer](https://user-images.githubusercontent.com/60596353/223839870-bcf4e784-ac6d-49ba-8d32-8693233d5311.jpg)

Have you cleared cache?

**Comment by huteck** [comment]
Hi, sorry I could not respond earlier unexpectedly.

I only tested it earlier locally with XAMPP so have now created a new grav with admin instance (fresh from Zip) online. Only installed future2021 theme via admin panel directly and added the social icon. Only visible as 16x16 hyperlink but without icon now.
As it is not a public facing folder yet, could you send me a way to ping you User/password valid for that verification period? Github seems not to allow private messages.

The future2021.yaml in the user/config/themes folder contains the following:
enable_social: true
social:
  -
    name: Instagram
    url: 'https://www.instagram.com/.../'
    icon: 'fa fa-user'

Or do I need to install/do anything else that the Fonts Awesome icons work? Are they locally or from Google API?

Thorben

**Comment by pmoreno-rodriguez** [comment]
Hello.
Having read your comment, I still don't understand what you want to achieve and what is the problem.

Could you indicate in two steps, 1, what you want to achieve and 2, what problem do you have for not achieving it?
Can you point me to an online site where I can see the error?

**Comment by huteck** [comment]
Sorry, I only was able to upload it to a webspace now. It's a fresh grav with admin installation + future2021 theme. I only added one social icon for showcase. 
![image](https://user-images.githubusercontent.com/117048767/230495538-a7f14cf6-dee3-4e7f-b57c-29f98931a132.png)

In the red arrow area, you will see that the curser changes and it highlights in the bottom left the URL, but the icon is not visible.

![image](https://user-images.githubusercontent.com/117048767/230495152-5a4b0c21-323a-4bb3-961e-cb464d954ab9.png)

![image](https://user-images.githubusercontent.com/117048767/230495706-cbd2465b-1347-4d8e-bb77-a20b3e466c79.png)

**Comment by huteck** [comment]
Forgot to mention, I installed the theme via the admin UI catalog. Not sure if there is a difference to the version you unser to replicate

**Comment by huteck** [comment]
Found now out a curiosity. I copied over your files from the demo folder to another installation. After saving (as enable_social is missing in the yaml file) it shows the icons properly. When I now click the "FontAwesome Icons" button and select a different one (e.g. fa fa-heart) it disappears. Could it be that there is an issue with getting the Icons on the page or the list has unsupported ones?

I confirmed this on above URL installation.  Selecting "fa fa-facebook" shows the icon while the one next to it with "fa fa-star" is not shown.

**Comment by pmoreno-rodriguez** [comment]
I'll take a look. It seems a bug.

**Comment by pmoreno-rodriguez** [comment]
The social template filters icons by brand class.
 (``` class="icon brands {{item.icon}}" ```). So, at the moment, you have to set these icons with a valid icon name from the [Brands Category](https://fontawesome.com/v5/search) .

Icon picker button show all Font awesome icons, not only brands icons.

I'll fix this issue in the next version.

**Comment by pmoreno-rodriguez** [comment]
Solved in new version 1.0.6
